Jenkinsfile

```diff
@@ -11,19 +11,25 @@ pipeline {
 		maven 'apache-maven-latest'
 		jdk 'openjdk-jdk17-latest'
 	}
+	environment {
+        SONARCLOUD_TOKEN     = credentials('SonarCloud token for gef')
```

Comment:
I would suggest loading the credentials via [withCredentials()](https://www.jenkins.io/doc/pipeline/steps/credentials-binding/) in the `Build` stage

i.e.
```
steps {
  withCredentials([string(credentialsId: 'sonarcloud-token', variable: 'SONARCLOUD_TOKEN')]) {
    withSonarQubeEnv(credentialsId: 'sonarcloud-token', installationName: 'GEF Classic') {
      ...
    }
  }
}
```

Reply:
@ptziegler as the latest update did also not work should I remove the environment section?

Reply:
@azoitl Yeah. Now that the variable is already defined in the stage, this entry seems obsolete.